apps/challenges/aws_utils.py

```diff
@@ -663,9 +662,6 @@ def scale_resources(challenge, worker_cpu_cores, worker_memory):
         ):
             challenge.task_def_arn = None
             challenge.save()
-        else:
-            response["Error"] = "Scaling inactive workers not supported"
-            return response
     except ClientError as e:
         logger.exception(e)
         return e.response
```

Comment:
Sam here send a dict with `"error": True, "message": "Scaling inactive workers not supported"`

Reply:
Done (I'm assuming that this is for the logger)